paddle/pybind/protobuf.cc

```diff
@@ -116,6 +117,11 @@ void BindProgramDesc(py::module &m) {
                   py::return_value_policy::reference)
       .def("append_block", &ProgramDescBind::AppendBlock,
            py::return_value_policy::reference)
+      .def("backward",
```

Comment:
I think `append_backward` is a better name. `backward` means the process that executes gradient op one by one, not append them.

Reply:
done